### DISCUSSION TITLE
[ERR] failed to load provider keys got EOF

### DISCUSSION THREAD

**Discussion by rachellechoufani** [projectdiscovery/nuclei#4837] [Q&A] [answered]
I am receiving this error whenever i run nuclei -h
[ERR] failed to load provider keys got EOF

**Comment by olearycrew** [comment]
I'm not able to find that error in the code base - what version of nuclei are you running?

Also I'm thinking that the "provider keys" reference might be to your nuclei config? If you back up and save your config and let nuclei rebuild it, does that fix the error (typically in `~/.config/nuclei/config.yaml`

**Reply by rachellechoufani** [reply]
It did not work. I am running nuclei v3.1.10 with nuclei-templates version: v9.7.7 (latest)
I tried uninstalling nuclei and reinstalled, the error persisted whenever i run any nuclei command.

**Reply by rachellechoufani** [reply]
Hello again.
I have just tested Nuclei on my local Windows machine, and I did not encounter this error. The commands executed smoothly without any issues.
However, the error persists in the virtualized environment Ubuntu Linux VM which was created using virtual box. 
Could there be issues with Nuclei on virtualized platforms like VirtualBox?

**Reply by olearycrew** [reply, marked as answer]
I believe it is probably related to a permissions issue on your Linux VM. Many folks run nuclei in virtualized platforms - but you have to make sure your user has the right permissions.

**Reply by hiteshmittal2020** [reply]
how did you uninstalled?

**Comment by GeorginaReeder** [comment]
Hi @hiteshmittal2020 - what are you trying to uninstall exactly?
